### PR TITLE
Rephrasing documentation for clariry.

### PR DESCRIPTION
**Pull request by letItCurl**
Hi all!

I have rephrased the documentation to emphasize the installation process. 
Please let me know if you find it more suitable.

RL

### CODE REVIEW

**Comment by letItCurl** [comment]
doc_reword

**Review by nic-agoda — APPROVED** [review]

**Comment by letItCurl** [comment]
Hi @virasak, what is remaining for me to get approval from a maintainer? :)

File: README.md [hunk 1]
```diff
@@ -1,56 +1,77 @@
 # Dotnet Build Metrics
 
-This is a custom MSBuild task that will measure the time to build each project and publishes the information to a datastore. The intent is to measure and improve developer experience on local workstations.
+## Overview
 
-**_Username and hostname are collected as a part of build information_**
+Dotnet Build Metrics is a custom MSBuild task that measures the time to build each project and publishes the information to a datastore. The intent is to measure and improve developer experience on local workstations.
```
Comment (dicko2) [inline README.md:5]: I think it's gone beyond this. We also have Web server startup time and the test data collection in the repo now.

I think the "build time" I would rephrase (for dotnet) to "the F5 experience". Because it's commonly known to dotnet developers that F5 is the should cut ket to debug your application.

The other part is test run data collection.

Overall the goal is around measuring and improving developer local experience

Comment (letItCurl) [reply README.md:5]: I see, It feels like the base work need more than just rephrasing.
Let me rework that.

**Comment by letItCurl** [comment]
@dicko2,

Updated the documentation using your input.

Hope like it :)
